pkg/node/node.go

```diff
@@ -255,6 +270,53 @@ func (n Node) removeLabel(key string) error {
 	return nil
 }
 
+// TaintSpotItn adds the spot termination notice taint onto a node
```

Comment:
Since this could be a breaking change for users upgrading, I think we should also add a flag in config (`--taint-node`) that defaults to false.